releasenotes/notes/0.19/add-pulse-gate-pass-dc347177ed541bcc.yaml

```diff
@@ -0,0 +1,100 @@
+---
+features:
+  - |
+    A new transpiler pass group ``qiskit.transpiler.passes.calibration`` has
+    been added.  This group consists of
+
+    - :class:`~qiskit.transpiler.passes.RZXCalibrationBuilder`
+    - :class:`~qiskit.transpiler.passes.RZXCalibrationBuilderNoEcho`
+    - :func:`~qiskit.transpiler.passes.calibration.rzx_templates.rzx_templates`.
+    - :class:`~qiskit.transpiler.passes.PulseGates`
+
+    :obj:`.RZXCalibrationBuilder`, :obj:`.RZXCalibrationBuilderNoEcho`, and
+    ``rzx_templates`` were moved from ``qiskit.transpiler.passes.scheduling``,
+    whereas :obj:`.PulseGates` is the new pass that automatically extracts
+    user-provided calibrations from the instruction schedule map and attaches
+    the gate schedule to the given (transpiled) quantum circuit as a pulse gate.
+    Any transpiler pass that interacts with the pulse gate feature will be
+    grouped to ``qiskit.transpiler.passes.calibration``.
+
+    This pass is applied to all optimization levels from 0 to 3.
+    No gate implementation is updated unless the end-user explicitly overrides
+    the ``backend.defaults().instruction_schedule_map``.
+
+    This pass saves users from individually calling
+    :meth:`.QuantumCircuit.add_calibration` for every circuit run on the
+    hardware.
```

Comment:
It is `PulseGates`. See here: https://github.com/Qiskit/qiskit-terra/blob/1a9f93804f2696e3a6c119ecbebd3724df008dfa/qiskit/transpiler/preset_passmanagers/level0.py#L262

Reply:
We should call that out explicitly then, or realistically do that and make this a separate bullet point because it's really a separate release note than the first part here.

Reply:
I had a bit of a rewrite in a40c877, but I may have got some of that not 100% correct. Would be good if someone with more knowledge could check.